test/e2e/streaming_e2e_test.go

```diff
@@ -0,0 +1,184 @@
+package e2e
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"net/http"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/livepeer/go-livepeer/eth"
+	lpTypes "github.com/livepeer/go-livepeer/eth/types"
+	"github.com/livepeer/go-livepeer/pm"
+	"github.com/stretchr/testify/require"
+)
+
+func TestE2EStreamingWorkflow(t *testing.T) {
+	geth := setupGeth(t)
+	defer terminateGeth(t, geth)
+
+	// 1. Start and register Orchestrator
+	o := startOrchestrator(t, geth)
+	// 2. Start Broadcaster
+	b := startBroadcaster(t, geth)
+	oEth := o.dev.Client
+	bEth := b.dev.Client
+	defer o.stop()
+	defer b.stop()
+	<-o.ready
+	<-b.ready
+
+	// when
+	// 1. Start and register Orchestrator
+	registerOrchestrator(o)
```

Comment:
```suggestion
	registerOrchestrator(o)
```

nit: the same here, this comment actually is even misleading because it says `Start`, but you're not starting anything here.